onnxruntime/core/session/onnxruntime_c_api.cc

```diff
@@ -843,6 +843,28 @@ void CheckAndAdjustInputSpansForLora(const OrtRunOptions& run_options,
 
 }  // namespace
 
+ORT_API_STATUS_IMPL(OrtApis::SetEpDynamicOptions, _In_ OrtSession* sess, _In_ const char** keys,
```

Comment:
ORT_API_STATUS_IMPL(SetEpDynamicOptions, _Inout_ OrtSession* sess, In_reads_(kv_len) const char* const* keys, In_reads_(kv_len) const char* const* values, In size_t kv_len)